GameWorld/View3D/SceneNodes/Rmv2MeshNode.cs

```diff
@@ -69,7 +69,7 @@ void UpdateMatrix()
         private Rmv2MeshNode()
         { }
 
-        public Rmv2MeshNode(RmvCommonHeader commonHeader, MeshObject meshObject, IMaterial material, AnimationPlayer animationPlayer, RenderEngineComponent renderEngineComponent, PbrShader shader = null)
+        public Rmv2MeshNode(RmvCommonHeader commonHeader, MeshObject meshObject, IMaterial material, AnimationPlayer animationPlayer, RenderEngineComponent renderEngineComponent, PbrShader shader = null )
```

Comment:
When possible, try to undo changes to files that you have edited but later changes. 
This should ideally not be part of the PR as it takes time and focus away from actual changes. 
Dont worry about it, just a note for later